python/examples/README.md

```diff
@@ -0,0 +1,358 @@
+
+The ones in `tmp/unpacked/bin`, however, will need the files from `tmp/unpacked/usr/local/lib` to work. These files include a `python311.zip` archive of the standard libraries, a placeholder `python3.11/lib-dynload` and a `python3.11/os.py`. The last two are not strictly necessary but if omitted will cause dependency warnings whenever python runs.
+
+```shell-session
+tmp
+├── [ 12M]  python-3.11.1-wasmedge.wasm
+├── [ 19M]  python-3.11.1.wasm
+└── [4.0K]  unpacked
+    ├── [4.0K]  bin
+    │   ├── [7.6M]  python-3.11.1-wasmedge.wasm
+    │   └── [ 14M]  python-3.11.1.wasm
+    └── [4.0K]  usr
+        └── [4.0K]  local
+            └── [4.0K]  lib
+                ├── [4.0K]  python3.11
+                │   ├── [4.0K]  lib-dynload
+                │   └── [ 39K]  os.py
+                └── [3.9M]  python311.zip
+```
+
+# First time running python.wasm
+
+Running the packed binaries is as easy as
+
+```shell-session
+wasmtime run \
+  tmp/python-3.11.1.wasm \
+  -- -c "import sys; from pprint import pprint as pp; \
+         pp(sys.path); pp(sys.platform)"
+['',
+ '/usr/local/lib/python311.zip',
+ '/usr/local/lib/python3.11',
+ '/usr/local/lib/python3.11/lib-dynload']
+'wasi'
+```
+
+To use the unpacked version we will need to pre-open the `usr/local/lib` folder as relative to the root path `/`, because `python.wasm` is compiled to look for it there. For Wasmtime this is done via `--mapdir`.
+
+```shell-session
+wasmtime run \
+  --mapdir /::$PWD/tmp/unpacked/ \
+  tmp/unpacked/bin/python-3.11.1.wasm \
+  -- -c "import sys; from pprint import pprint as pp; \
+         pp(sys.path); pp(sys.platform)"
+
+['',
+ '/usr/local/lib/python311.zip',
+ '/usr/local/lib/python3.11',
+ '/usr/local/lib/python3.11/lib-dynload']
+'wasi'
+```
+
+We could do the same with the WasmEdge-compliant binary (note the slight differences in the CLI arguments).
+
+```shell-session
+wasmedge \
+  --dir /:$PWD/tmp/unpacked/ \
+  tmp/unpacked/bin/python-3.11.1-wasmedge.wasm \
+  -c "import sys; from pprint import pprint as pp; \
+      pp(sys.path); pp(sys.platform)"
+
+['',
+ '/usr/local/lib/python311.zip',
+ '/usr/local/lib/python3.11',
+ '/usr/local/lib/python3.11/lib-dynload']
+'wasi'
+```
+
+## Running the repl
+
+If you want, you can play with the Python repl.
+
+```shell-session
+wasmtime run tmp/unpacked/bin/python-3.11.1.wasm
+
+Python 3.11.1 (tags/v3.11.1:a7a450f, Feb 17 2023, 12:59:00) ... on wasi
+Type "help", "copyright", "credits" or "license" for more information.
+>>> import sys
+>>>
+>>> sys.platform
+'wasi'
+>>>
+>>> sys.version_info
+sys.version_info(major=3, minor=11, micro=1, releaselevel='final', serial=0)
+```
+
+## Running an app with dependencies
+
+Next, let's assume we have a Python app that has additional dependencies. For example [emojize_text.py](./workdir/emojize_text.py).
+
+### Installing dependencies to the pre-compiled PYTHONPATH
+
```

Comment:
I think we could add a disclaimer note here explaining not all packages are available yet... (ie: NumPy)